docs/en/advanced/Tiered-Locality.md

```diff
@@ -6,6 +6,12 @@ group: Advanced
 priority: 1
 ---
 
+TODO(andrew): Add intro section which talks about why users would want to use this feature (specific use cases)
+TODO(andrew): Include example(s) of how to verify that tiered locality is set up correctly
+TODO(andrew): Focus on setting tiered locality via script, only discuss setting via configuration towards the end
```

Comment:
is one a basic setup and the other an advanced setup?

Reply:
In most situations, setting via script should be easier since you can use the same script on all nodes. I think the complexity of the two approaches is similar, but I'd like to make one the recommended/default so that new users reading this doc aren't confused about which way they should set things.